src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -359,8 +359,8 @@ class FileField extends React.Component {
                     )}
                   {!file.uploading &&
                     hasErrors && (
-                      <span className="usa-input-error-message">
-                        {errors[0]}
+                      <span className="usa-input-error-message" role="alert">
+                        <span className="sr-only">Error</span> {errors[0]}
```

Comment:
Tagging @1Copenut to give some guidance here.

I followed the ticket and then went to a page on [WCAG techniques](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19). What I'm curious about is:

> The error container must be present in the DOM on page load for the error message to be spoken by most screen readers.

Is this solution going to work considering that the span is gated behind `hasErrors`?

Reply:
@dat-gcio & @bkjohnson Our forms library adds the `span[role="alert"]` message dynamically after the page has rendered, and we get good, consistent responses from screen readers. I'd propose we go forward with this code pattern and test in staging. We can always iterate if it's not behaving as expected.

I put together this pen ( https://codepen.io/tpierce_402/pen/Lqezye ) some time ago because IE11 + JAWS was reading the error message twice. It's not quite the same thing, but I built up the error message to be injected on the fly, and removed when the error was gone, just like you have here.